README.md

```diff
@@ -2,16 +2,57 @@
 
 An easy to use client for the [Flagr](https://checkr.github.io/flagr) feature flag service.
 
+## General Concepts used in this documentation
+ * **Flag** can be a feature flag, an experiment, or a configuration.
+ * **Context** is the information that needs to be provided to evaluate a **flag**.
+ * **Variant** represents the possible variation of a flag.
+ * **Variant Attachment** represents dynamic properties that can be added to the **context**. With this it's possible to rules on which **variant** to return.
```

Comment:
This is not correct. The variant attachment is additional meta information that you can return with your variant.

Reply:
fixed